www/src/templates/template-docs-markdown.js

```diff
@@ -84,6 +85,25 @@ class DocsTemplate extends React.Component {
                 {page.frontmatter.title}
               </h1>
               <div
+                css={{
+                  "> .gatsby-code-title": {
+                    backgroundColor: colors.gatsby,
+                    borderTopLeftRadius: 8,
+                    borderTopRightRadius: 8,
+                    color: `white`,
+                    fontFamily:
```

Comment:
you can also get our font families from https://github.com/gatsbyjs/gatsby/blob/c87e937182589c4ddc07ffd5ba9dadba94c88cc7/www/src/utils/typography.js#L30 e.g. `typography.options.monospaceFontFamily.join(",")`